style-dictionary/classic/metadata.ts

```diff
@@ -13,6 +13,7 @@ const updatedDescriptions: StyleDictionary.TokenIndex<string> = {
     'The color of clickable elements in their default state. *For example: expandable sections, tabs, and icons.*',
   colorTextInteractiveHover:
     'The color of clickable elements on hover. *For example: expandable sections, and icons on hover.*',
+  colorTextInteractivePressed: 'The color of clickable elements in pressed state.',
```

Comment:
Adding this here will make this a public token. Just to make sure, is this intentional?